README.md

```diff
@@ -40,6 +40,31 @@ From your `SharedModule`:
 export class SharedModule {}
 ```
 
+Style the component the way you want (example):
+```diff
++ngx-row-accordion {
```

Comment:
This would actually need to be `ngx-row-accordion ::ng-deep` for most people's use cases

Reply:
It might be worth changing it to that to avoid confusion, or explain that when in view encapsulated that is needed, otherwise in main.scss etc it would not be needed.

Reply:
Damn my page didn't reload so I merged before I saw your comment.

Opening a new one to fix that :+1:

Reply:
Follow up https://github.com/maxime1992/ngx-row-accordion/pull/2